spark-jobs/src/main/scala/filodb/downsampler/chunk/BatchExporter.scala

```diff
@@ -311,11 +329,12 @@ case class BatchExporter(downsamplerSettings: DownsamplerSettings, userStartTime
       val metric = labels(LABEL_NAME)
       // to compute YYYY, MM, dd, hh
       // to compute readable timestamp from unix timestamp
-      val timestamp = Instant.ofEpochMilli(epoch_timestamp).atZone(ZoneId.systemDefault()).toLocalDateTime()
-      val year = timestamp.getYear()
-      val month = timestamp.getMonthValue()
-      val day = timestamp.getDayOfMonth()
-      val hour = timestamp.getHour()
+      val dateTime = Instant.ofEpochMilli(epoch_timestamp).atZone(ZoneId.systemDefault()).toLocalDateTime()
+      val timestamp = java.sql.Timestamp.valueOf(dateTime)
```

Comment:
nitpick - add a `IMPORTANT`  comment since this is required by Spark. Don't want people updating it accidentally

Reply:
Comment added ✅